src/components/header/sign-markup.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { render, screen } from '@testing-library/react';
+import { withHistory, withStore } from '../../mock-component';
+import { AuthorizationStatus } from '../../constants';
+import SignMarkup from './sign-markup';
+
+describe('Component: SignMarkup', () => {
+  const signMarkupContainerTestId = 'auth-markup-container';
+  const unsignMarkupContainerTestId = 'no-auth-markup-container';
+
+  it('should render correct when user not authorized', () => {
+    const { withStoreComponent } = withStore(<SignMarkup />, {
+      AUTH: {
+        authStatus: AuthorizationStatus.NoAuth,
+      }
+    });
+    const withHistoryComponent = withHistory(withStoreComponent);
+
+    render(withHistoryComponent);
+    const SignMarkupContainer = screen.queryByText(signMarkupContainerTestId);
+    const UnsignMarkupContainer = screen.getByTestId(unsignMarkupContainerTestId);
+
+    expect(SignMarkupContainer).not.toBeInTheDocument();
+    expect(UnsignMarkupContainer).toBeInTheDocument();
+  });
+
+  it('should render correct when user authorized', () => {
+    const { withStoreComponent } = withStore(<SignMarkup />, {
+      AUTH: {
+        authStatus: AuthorizationStatus.Auth,
+      }
+    });
+
+    const withHistoryComponent = withHistory(withStoreComponent);
+
+    render(withHistoryComponent);
+    const SignMarkupContainer = screen.getByTestId(signMarkupContainerTestId);
+    const UnsignMarkupContainer = screen.queryByText(unsignMarkupContainerTestId);
```

Comment:
И везде, где это повторяется, тоже надо изменить

```suggestion
    const signMarkupContainer = screen.getByTestId(signMarkupContainerTestId);
    const unsignMarkupContainer = screen.queryByText(unsignMarkupContainerTestId);
```